AEPTarget/Tests/FunctionalTests/TargetFunctionalTests.swift

```diff
@@ -262,8 +262,6 @@ class TargetFunctionalTests: TargetFunctionalTestsBase {
         target = Target(runtime: mockRuntime)
         target.onRegistered()
         XCTAssertEqual("935CDD24-8FD7-4B30-8508-4BE40C3FC263", target.targetState.storedSessionId)
-        let sessionId = target.targetState.sessionId
```

Comment:
This call would generate a new session Id as session expired (see L258 where timestamp is set to a year back in setup) and update the session timestamp so when edge host is queried for the prefetch request, the new session wouldn't timeout and it would still be used. However, the resetting the Edge host on launch where there is no prev stored configuration masked this.